samples/AppSecretsConfig/Pages/Index.cshtml

```diff
@@ -0,0 +1,38 @@
+@page
```

Comment:
Consider dropping razor/css/js and doing a simple 

```
  app.Run(async context =>
        {
            await context.Response.WriteAsync(....);
        });
```

Reply:
I started from the `dotnet new webapp` template and only pruned a couple things, but Razor was introduced with Visual Studio 2010 and should be supported on any platform we support, yes? The main goal of the sample was to show how to configure an ASP.NET application, and that may be limited to ASP.NET Core or at least newer .NET Framework versions that use the "new" configuration provider - certainly much different than my more active days with ASP.NET Classic using `System.Configuration` (way easier and more powerful now than back then).

Reply:
>should be supported on any platform we support, yes? 

It is supported yes. But in ASP.NET Core patterns change and things get obsolete. And it doesn't matter to the goal of the sample whether we use razor or not. 

For example, I had issues with  when trying to migrate https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Integration.Tests.AspNetCoreWebApp to net5

Reply:
Understood, but at the same time I was trying to have something similar to what users might be using which is likely to use some sort of middleware, whether Razor, Blazor, etc.